frame/bags-list/src/tests.rs

```diff
@@ -196,6 +196,284 @@ mod pallet {
 			assert_storage_noop!(assert!(BagsList::rebag(Origin::signed(0), 10).is_ok()));
 		})
 	}
+
+	#[test]
+	fn put_in_front_of_two_node_bag_heavier_is_tail() {
+		ExtBuilder::default().add_ids(vec![(10, 15), (11, 16)]).build_and_execute(|| {
+			// given
+			assert_eq!(
+				List::<Runtime>::get_bags(),
+				vec![(10, vec![1]), (20, vec![10, 11]), (1_000, vec![2, 3, 4])]
+			);
+
+			StakingMock::set_vote_weight_of(&10, 15);
```

Comment:
I think add_ids should also set this value. More specifically, where we in `fn build` of `ExtBuilder` we should update this new hash map all the time. Any exception should be handled by `set_vote_weight_of `

Reply:
also, it would be cleaner to add a `fn initial_voters(&mut self, has: bool) -> Self ` to `ExtBuilder` and if you set it to `false`, then `GENESIS_IDS` get skipped. Most of the new tests only care about one bag.

Reply:
387ab0dad45da9500de0b6f8efb2323718f1d777 for the first comment

Reply:
7d5a1ca3ecd92be266b8c89060950fbb041c38da adds `skip_genesis_ids`